salt/orch/update.sls

```diff
@@ -60,6 +60,7 @@ update_modules:
 {{ master_id }}-wait-for-start:
   salt.wait_for_event:
     - name: salt/minion/*/start
+    - timeout: 1200
```

Comment:
Sadly, last time I tried to access the pillar here, it wouldn't work - so I went with a long enough period that anything should be able to reboot, but not so long people will be waiting hours to see the failure.